src/Buffer.ts

```diff
@@ -34,6 +34,9 @@ export default class Buffer extends events.EventEmitter {
         var current = {attributes: <i.Attributes>null, text: ''};
         var cursorPosition = cursor.getPosition();
 
+        /* Quick fix for the strange character at the end (https://github.com/black-screen/black-screen/pull/56#issuecomment-139653723) */
+        if (process.platform === 'win32' && row.length < 1) return;
```

Comment:
It's not a mistake to use strict equality (triple equals) in TypeScript. When you read a piece of code that contains strict equality it's obvious to the reader that no type coercion will happen there. TypeScript transpiles equality operators as is to the JavaScript code, so the meaning is still clear in js.
Also if you take a look at the official [TypeScript samples](http://www.typescriptlang.org/Samples), you will see they almost always use strict equality in their ts files (e.g. [raytracer.ts](https://github.com/Microsoft/TypeScriptSamples/blob/master/raytracer/raytracer.ts)).